config/samples/v1alpha1_aigateway_test.yaml

```diff
@@ -0,0 +1,21 @@
+---
```

Comment:
The idea with having it in samples is to have *one* sample that can be used both for the test and for manual evaluation. Now, we still have duplications.